From e91335b95ff98b38bfa8f054fda9ff8f4abdae87 Mon Sep 17 00:00:00 2001
From: Alex Williamson <alex.williamson@redhat.com>
Date: Thu, 7 Aug 2014 21:02:57 +0200
Subject: [PATCH 3/7] vfio: correct debug macro typo

Message-id: <20140807210257.11689.30368.stgit@gimli.home>
Patchwork-id: 60479
O-Subject: [RHEL7.0/z qemu-kvm PATCH v2 2/6] vfio: correct debug macro typo
Bugzilla: 1098976
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: Bandan Das <bsd@redhat.com>

Change to DEBUG_VFIO in vfio_msi_interrupt() for debug
messages to get printed

Signed-off-by: Bandan Das <bsd@redhat.com>
Signed-off-by: Alex Williamson <alex.williamson@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/misc/vfio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/misc/vfio.c b/hw/misc/vfio.c
index b5387ca..7f6ebf9 100644
--- a/hw/misc/vfio.c
+++ b/hw/misc/vfio.c
@@ -619,7 +619,7 @@ static void vfio_msi_interrupt(void *opaque)
         return;
     }
 
-#ifdef VFIO_DEBUG
+#ifdef DEBUG_VFIO
     MSIMessage msg;
 
     if (vdev->interrupt == VFIO_INT_MSIX) {
-- 
1.8.3.1

